oncall/autoIssueAssign.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+autoAssign() {
+  curl -X POST "https://api.github.com/repos/microsoft/Oryx/issues/$3/assignees" -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $2" -d "{\"assignees\": ["\"$1\""]}"
```

Comment:
We should check the response status of this `POST` request since we're just shooting the request off and then assuming it worked based on the calls to this function down below. There may be an issue with the token being expired or an invalid user being provided from one of our Variables, so it would be best to check if the response of this `POST` was a `201`, log an error if not, and return an appropriate value from this function to signal to the caller if the request was successful or not.